server/tables/tables.go

```diff
@@ -225,7 +226,7 @@ func (u *User) ToProto() *uspb.DisplayUser {
 			Id: u.UserID,
 		},
 		Name: &uspb.Name{
-			Full:  u.FirstName + " " + u.LastName,
+			Full:  strings.TrimSpace(u.FirstName + " " + u.LastName),
```

Comment:
nit: should this be `TrimSpace(firstName) + " " + TrimSpace(lastName)` ?

Reply:
The only thing I'm trying to achieve here is remove the space in case FirstName or LastName (or neither) are set.